src/App/App.js

```diff
@@ -0,0 +1,19 @@
+import './App.css';
+import React from 'react';
+import { NavBar } from '../NavBar/NavBar';
+import { Strains } from '../Strains/Strains';
+// import Experiences from '../Experiences/Experiences';
+import { Route, Switch, Redirect } from 'react-router-dom';
+
+export const App = () => {
+  return (
+    <section>
+      <NavBar />
+      <Switch>
+        <Redirect exact path="/" to="/strains" component={Strains} />
```

Comment:
what's going on here? if you have just one endpoint '/strains' then just use that one route. if you have 2, order them from most specific to least, and you don't need an exact at that point